src/custom-element.js

```diff
@@ -1800,6 +1800,11 @@ function createBaseCustomElementClass(win, elementConnectedCallback) {
     mutatedAttributesCallback(mutations) {
       if (this.impl_) {
         this.impl_.mutatedAttributesCallback(mutations);
+      } else if (this.R1()) {
+        // If amp-bind is trying to mutate an uninitialized BaseElement, it is probably about time
+        // to initialize it.
+        const scheduler = getSchedulerForDoc(this);
+        scheduler.scheduleAsap(this);
```

Comment:
How's this different than a component setting `deferredMount = false` when it needs to mutate before mount?

Reply:
First, this affects every component instead of only a specific one.

Second, the mount of the component will now be deferred until the amp-bind event occurs instead of on document load. Especially for an initially hidden component it may make sense to defer initialization.

Reply:
Would this allow us to remove `deferredMount = false` from the components that define it?